cczoo/rag/build-images.sh

```diff
@@ -1,12 +1,27 @@
 #!/bin/bash
 set -ex
 
+dcap_version=""
+
+while [[ "$#" -gt 0 ]]; do
+    case $1 in
+        --dcap-version) dcap_version="$2"; shift ;;
+        *) echo "Unknown parameter passed: $1"; exit 1 ;;
+    esac
+    shift
+done
+
+if [ -z "$dcap_version" ]; then
+    echo "Error: --dcap-version is required"
```

Comment:
when dcap_version is not provided, could suggest user to check version in below link:
https://download.01.org/intel-sgx/sgx-dcap/